core/src/test/scala/com/yahoo/maha/core/query/DefaultQueryPipelineFactoryTest.scala

```diff
@@ -501,17 +501,23 @@ class DefaultQueryPipelineFactoryTest extends FunSuite with Matchers with Before
         row.addValue("Ad Group ID", 202)
         row.addValue("Impressions", 100)
         row.addValue("Clicks", 1)
+        row.addValue("Pricing Type", "CPC")
```

Comment:
Please create separate test for new functionality.  Should be easy to copy/paste the parts you need.